build_python_framework_pkgs.zsh

```diff
@@ -18,6 +18,10 @@ RP_BINDIR="/tmp/relocatable-python"
 MP_BINDIR="/tmp/munki-pkg"
 CONSOLEUSER=$(/usr/bin/stat -f "%Su" /dev/console)
 PIPCACHEDIR="/Users/${CONSOLEUSER}/Library/Caches/pip"
+# NOTARY_PASS="" # Store as a repo secret
+XCODE_PATH="/Applications/Xcode_13.2.1.app"
```

Comment:
Can we use a variable from inside GHA instead? If they change the xcode version, this will instantly break.

Reply:
They maintain several versions of Xcode and symbolic link to Xcode.app for their "stable paths". Even if it breaks in the future, I think we should "pin" to the one we know works until we've tested a new one.